packages/xod-client/src/core/styles/components/ReactDatasheet.scss

```
@@ -0,0 +1,97 @@
+span.data-grid-container, span.data-grid-container:focus {
+  outline: none;
+}
+
+.data-grid-container .data-grid {
+  table-layout: fixed;
+  border-collapse: collapse;
+}
+
+.data-grid-container .data-grid .cell.updated {
+    background-color: rgba(0, 145, 253, 0.16);
+    transition: background-color 0ms ease ;
+}
+
+.data-grid-container .data-grid .cell {
+  box-sizing: border-box;
+  user-select: none;
+  cursor: cell;
+  background-color: $dark-light;
+  color: $input-color-text;
+  transition: background-color 100ms ease;
+  vertical-align: middle;
```

```suggestion
  vertical-align: middle;
  line-height: 1.6;
```
It provides better vertical aligning. What do you think?